http/router/fs.lua

```diff
@@ -62,14 +63,14 @@ local function static_file(self, request, format)
         }
     end
 
-    local s, fh = pcall(io.input, file)
+    local fh, _ = fio.open(file, {'O_RDONLY'})
 
-    if not s then
+    if fh == nil then
         return { status = 404 }
     end
 
-    local body = fh:read('*a')
-    io.close(fh)
+    local body = fh:read()
```

Comment:
I think we should check errors in all cases and handle them properly. Not just ignore.